tinygrad/runtime/ops_metal.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, name:str, prg:str):
       self.library = unwrap(METAL.device.newLibraryWithData_error_(data, None))
     else:
       options = Metal.MTLCompileOptions.alloc().init()
+      options.setFastMathEnabled_(0)
```

Comment:
Any way to be more granular with this? I worry this'll make metal slower

Reply:
Looking at the docs, I don't see any options to narrow this to only non-NaN optimizations like in the LLVM case.